### PR TITLE
User Access and misc UXA

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#17464
/bcgov/entity#16954

*Description of changes:*
* Minor UXA updates for User Access Flow
* Prevent pre-populate of Authorizer for LawyerNotary
* Wider styling adjustment to 'Incomplete Step' messaging 
* Minor alignment adjustments to user access flow review
* Removed obsolete FF

Screenshot 2024-01-18 at 11 34 23 AM

Screenshot 2024-01-18 at 11 34 12 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/utils/feature-flags.ts [hunk 1]
```diff
@@ -11,8 +11,7 @@ export const defaultFlagSet: LDFlagSet = {
   'search-serial-number': true,
   'mhr-ui-enabled': true, // Mhr Search - default true: Should remove from codebase
   'mhr-registration-enabled': false, // Enables MHR table tab
-  'mhr-transfer-enabled': false, // Enables changes to base MHR Home Owners within the MHR Information flow
-  'assets-tiptap-enabled': true, // Enables new TipTap wysiwyg editor - default true: Should remove from codebase
```
Comment (cameron-eyds) [inline ppr-ui/src/utils/feature-flags.ts:15]: FF No longer required

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1695-55s02835.web.app

**Review by doug-lovett — APPROVED** [review]
Yahoo, more fun with UXA.

**Comment by cameron-eyds** [comment]
> Yahoo, more fun with UXA.

Much like golf, it's a lifelong goal, never really get there 100%.

**Pull request merged**